asmcli/commands/install.sh

```diff
@@ -86,6 +87,13 @@ install_private_ca() {
     --location "${CA_LOCATION}" \
     --member "group:${WORKLOAD_IDENTITY}" \
     --role "roles/privateca.auditor"
+
+  if [[ $(grep -o ":" <<< "${CA_NAME}" | wc -l) -eq 1 ]]; then
+    local CERT_TEMPLATE; CERT_TEMPLATE=$(echo "${CA_NAME}" | cut -f2 -d:)
+    retry 3 gcloud privateca templates add-iam-policy-binding "${CERT_TEMPLATE}" \
```

Comment:
Not sure how to check whether have the privileges to bind IAM roles, one thought here is that even it fails to bind the roles, it will not fail the installation. So from that perspective this is something user could work around.
But agreed on that is right things to do as following, if you know the command that check whether user have the permission to set the bindings, please let me know.

Reply:
A good compromise IMO would be to gate ALL the IAM role changes with the "can_modify_gcp_iam_roles" function. I think adding checks here could get complicated. How about we make the required IAM roles more easily available in our documentation?

Reply:
Added the conditional check "can_modify_gcp_iam_roles".